tests/unattended/unit-tests.sh

```diff
@@ -0,0 +1,197 @@
+trap clean SIGINT
+
+logfile="./unit-tests.log"
+echo "-------------------------" >> ./unit-tests.log
+debug=">> ${logfile}"
+ALL_FILES=("common" "checks")
+
+function logger() {
+
+    now=$(date +'%d/%m/%Y %H:%M:%S')
+    case ${1} in 
+        "-e")
+            mtype="ERROR:"
+            message="${2}"
+            ;;
+        "-w")
+            mtype="WARNING:"
+            message="${2}"
+            ;;
+        *)
+            mtype="INFO:"
+            message="${1}"
+            ;;
+    esac
+    echo "${now} ${mtype} ${message}" | tee -a ${logfile}
+
+}
+
+
+function createImage() {
+
+    if [ ! -f ./Dockerfile ]; then
+        logger -e "No Dockerfile found to create the environment."
+        exit 1
+    fi
+
+    image_name="testing-img"
+    if [ -z "$(docker images | grep $image_name)" ]; then
+        eval "docker build -t $image_name . ${debug}"
+        if [ "$?" != 0 ]; then
+            logger -e "Docker encountered some error."
+            exit 1
+        else 
+            logger "Docker image created successfully."
+        fi
+    else 
+        logger "Docker image found."
+    fi
+}
+
+function runContainer() {
+    container_name="testing-container"
+    eval "docker run -d -t --name $container_name $image_name /bin/bash ${debug}"
+    if [ "$?" != 0 ]; then
+        logger -e "Docker encountered some error."
+        exit 1
+    else 
+        logger "Docker container created successfully."
+    fi
+    container_id="$( docker ps -a | grep $container_name | awk '{ print $1 }' )"
+    eval "docker cp bach.sh $container_id:/tests/unattended/bach.sh ${debug}"
```

Comment:
Why do you use docker cp instead of copying bach.sh in the Docker building stage?

Reply:
Changed in https://github.com/wazuh/wazuh-packages/pull/1136/commits/306e7205e0eb91c86c5dc321022b65fdc26d6af6